### PR TITLE
Irminger 8 Updates

### PR DESCRIPTION
**Pull request by cdobs**
Updating Irminger 8 deploy csvs as things get deployed

### CODE REVIEW

File: deployment/GI05MOAS-GL469_Deploy.csv [hunk 1]
```diff
@@ -7,7 +7,7 @@ AR46,,AR46,,GI05MOAS-GL469-00-ENG000000,3,1,2020-08-15T17:21:00,2020-08-16T00:11
...
-AR60-01,,,,GI05MOAS-GL469-01-FLORDM000,4,1,2021-08-15T00:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00469,CGINS-FLORDM-03528,59.925833,-39.299167,,1000,1000,,,
-AR60-01,,,,GI05MOAS-GL469-02-DOSTAM000,4,1,2021-08-15T00:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00469,CGINS-DOSTAM-00168,59.925833,-39.299167,,1000,1000,,,
-AR60-01,,,,GI05MOAS-GL469-04-CTDGVM000,4,1,2021-08-15T00:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00469,CGINS-CTDGVM-09172,59.925833,-39.299167,,1000,1000,,,
\ No newline at end of file
+AR60-01,,,,GI05MOAS-GL469-00-ENG000000,4,1,2021-08-11T04:04:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00469,CGCON-GLDROO-00469,59.85902,-39.12405,,1000,1000,,,
+AR60-01,,,,GI05MOAS-GL469-01-FLORDM000,4,1,2021-08-11T04:04:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00469,CGINS-FLORDM-03528,59.85902,-39.12405,,1000,1000,,,
+AR60-01,,,,GI05MOAS-GL469-02-DOSTAM000,4,1,2021-08-11T04:04:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00469,CGINS-DOSTAM-00168,59.85902,-39.12405,,1000,1000,,,
+AR60-01,,,,GI05MOAS-GL469-04-CTDGVM000,4,1,2021-08-11T04:04:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00469,CGINS-CTDGVM-09172,59.85902,-39.12405,,1000,1000,,,
```
Comment (rtravis-whoi) [inline deployment/GI05MOAS-GL469_Deploy.csv:13]: looks good

File: deployment/GI05MOAS-GL537_Deploy.csv [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
-AR60-01,,,,GI05MOAS-GL537-01-FLORDM000,3,1,2021-08-15T00:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00537,CGINS-FLORDM-03582,59.925833,-39.299167,,1000,1000,,,
-AR60-01,,,,GI05MOAS-GL537-02-DOSTAM000,3,1,2021-08-15T00:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00537,CGINS-DOSTAM-00401,59.925833,-39.299167,,1000,1000,,,
-AR60-01,,,,GI05MOAS-GL537-04-CTDGVM000,3,1,2021-08-15T00:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00537,CGINS-CTDGVM-09237,59.925833,-39.299167,,1000,1000,,,
\ No newline at end of file
+AR60-01,,,,GI05MOAS-GL537-00-ENG000000,3,1,2021-08-11T04:10:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00537,CGCON-GLDROO-00537,59.85909,-39.12473,,1000,1000,,,
+AR60-01,,,,GI05MOAS-GL537-01-FLORDM000,3,1,2021-08-11T04:10:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00537,CGINS-FLORDM-03582,59.85909,-39.12473,,1000,1000,,,
+AR60-01,,,,GI05MOAS-GL537-02-DOSTAM000,3,1,2021-08-11T04:10:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00537,CGINS-DOSTAM-00401,59.85909,-39.12473,,1000,1000,,,
+AR60-01,,,,GI05MOAS-GL537-04-CTDGVM000,3,1,2021-08-11T04:10:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00537,CGINS-CTDGVM-09237,59.85909,-39.12473,,1000,1000,,,
```
Comment (rtravis-whoi) [inline deployment/GI05MOAS-GL537_Deploy.csv:5]: For Lat I get 59.85908333...so 59.85908. The rest looks good

File: deployment/GI05MOAS-PG565_Deploy.csv [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
-AR60-01,,,,GI05MOAS-PG565-06-PARADM000,2,1,2021-08-15T00:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00565,CGINS-PARADM-50198,59.925833,-39.299167,,200,200,,,
\ No newline at end of file
+AR60-01,,,,GI05MOAS-PG565-00-ENG000000,2,1,2021-08-11T03:11:00,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00565,CGCON-GLDRGP-00565,59.86468,-39.12545,,200,200,,,
+AR60-01,,,,GI05MOAS-PG565-01-CTDGVM000,2,1,2021-08-11T03:11:00,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00565,CGINS-CTDGVM-09285,59.86468,-39.12545,,200,200,,,
+AR60-01,,,,GI05MOAS-PG565-02-DOSTAM000,2,1,2021-08-11T03:11:00,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00565,CGINS-DOSTAM-00142,59.86468,-39.12545,,200,200,,,
+AR60-01,,,,GI05MOAS-PG565-03-FLORTM000,2,1,2021-08-11T03:11:00,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00565,CGINS-FLORTM-03813,59.86468,-39.12545,,200,200,,,
+AR60-01,,,,GI05MOAS-PG565-04-FLORTO000,2,1,2021-08-11T03:11:00,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00565,CGINS-FLORTO-01330,59.86468,-39.12545,,200,200,,,
+AR60-01,,,,GI05MOAS-PG565-05-NUTNRM000,2,1,2021-08-11T03:11:00,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00565,CGINS-NUTNRM-00730,59.86468,-39.12545,,200,200,,,
```
Comment (rtravis-whoi) [inline deployment/GI05MOAS-PG565_Deploy.csv:7]: Looks good.  Confirmed new NUTNR SN.

File: deployment/GI01SUMO_Deploy.csv [hunk 4]
```diff
@@ -1,391 +1,391 @@
...
+AR60-01,,,,GI01SUMO-SBD11-01-MOPAK0000,8,1,2021-08-12T17:04:00,,CGMGI-01SUMO-00008,,CGINS-MOPAK0-43264,59.946233,-39.57565,,0,2662,,,
+AR60-01,,,,GI01SUMO-SBD11-03-HYDGN0000,8,1,2021-08-12T17:04:00,,CGMGI-01SUMO-00008,,CGINS-HYDGN0-49876,59.946233,-39.57565,,0,2662,,,
+AR60-01,,,,GI01SUMO-SBD11-04-DOSTAD000,8,1,2021-08-12T17:04:00,,CGMGI-01SUMO-00008,,CGINS-DOSTAD-00460,59.946233,-39.57565,,1,2662,,,
+AR60-01,,,,GI01SUMO-SBD11-05-SPKIRB000,8,1,2021-08-12T17:04:00,,CGMGI-01SUMO-00008,,CGINS-SPKIRB-00270,59.946233,-39.57565,,-6,2662,,,
+AR60-01,,,,GI01SUMO-SBD11-06-METBKA000,8,1,2021-08-12T17:04:00,,CGMGI-01SUMO-00008,,CGINS-METLGR-00034,59.946233,-39.57565,,0,2662,,,
+AR60-01,,,,GI01SUMO-SBD11-08-NUTNRB000,8,1,2021-08-12T17:04:00,,CGMGI-01SUMO-00008,,CGINS-NUTNRB-01092,59.946233,-39.57565,,1,2662,,,
+AR60-01,,,,GI01SUMO-SBD12-00-DCLENG000,8,1,2021-08-12T17:04:00,,CGMGI-01SUMO-00008,,CGCON-BDCL12-50009,59.946233,-39.57565,,0,2662,,,
+AR60-01,,,,GI01SUMO-SBD12-01-OPTAAD000,8,1,2021-08-12T17:04:00,,CGMGI-01SUMO-00008,,CGINS-OPTAAD-00205,59.946233,-39.57565,,1,2662,,,
```
Comment (rtravis-whoi) [inline deployment/GI01SUMO_Deploy.csv:385]: buoy OPTAA 245 replaced with 205 - looks good

File: deployment/GI01SUMO_Deploy.csv [hunk 5]
```diff
@@ -1,391 +1,391 @@
...
+AR60-01,,,,GI01SUMO-SBD12-02-FLORTD000,8,1,2021-08-12T17:04:00,,CGMGI-01SUMO-00008,,CGINS-FLORTD-01116,59.946233,-39.57565,,1,2662,,,
+AR60-01,,,,GI01SUMO-SBD12-03-HYDGN0000,8,1,2021-08-12T17:04:00,,CGMGI-01SUMO-00008,,CGINS-HYDGN0-49891,59.946233,-39.57565,,0,2662,,,
+AR60-01,,,,GI01SUMO-SBD12-04-PCO2AA000,8,1,2021-08-12T17:04:00,,CGMGI-01SUMO-00008,,CGINS-PCO2AA-34232,59.946233,-39.57565,,0,2662,,,
+AR60-01,,,,GI01SUMO-SBD12-05-WAVSSA000,8,1,2021-08-12T17:04:00,,CGMGI-01SUMO-00008,,CGINS-WAVSSA-06421,59.946233,-39.57565,,0,2662,,,
+AR60-01,,,,GI01SUMO-SBD12-06-METBKA000,8,1,2021-08-12T17:04:00,,CGMGI-01SUMO-00008,,CGINS-METLGR-00036,59.946233,-39.57565,,0,2662,,,
+AR60-01,,,,GI01SUMO-SBD12-08-FDCHPA000,8,1,2021-08-12T17:04:00,,CGMGI-01SUMO-00008,,CGINS-FDCHPA-44904,59.946233,-39.57565,,-6,2662,,,
```
Comment (rtravis-whoi) [inline deployment/GI01SUMO_Deploy.csv:391]: Looks good.

**Review by rtravis-whoi — CHANGES_REQUESTED** [review]
Glider 537 - I get slightly different lat value.

File: deployment/GI05MOAS-PG565_Deploy.csv (hunk 3 above)
Comment (snwhite) [reply deployment/GI05MOAS-PG565_Deploy.csv:7]: Looks good.

File: deployment/GI05MOAS-GL537_Deploy.csv (hunk 2 above)
Comment (snwhite) [reply deployment/GI05MOAS-GL537_Deploy.csv:5]: Concur with @rtravis-whoi.  Perhaps the 59.95909 is the original from which 59 51.525 was calucated?

File: deployment/GI05MOAS-GL469_Deploy.csv (hunk 1 above)
Comment (snwhite) [reply deployment/GI05MOAS-GL469_Deploy.csv:13]: Looks good.

File: deployment/GI01SUMO_Deploy.csv (hunk 5 above)
Comment (snwhite) [reply deployment/GI01SUMO_Deploy.csv:391]: Looks good.

File: deployment/GI05MOAS-GL537_Deploy.csv [hunk 6]
```diff
@@ -1,5 +1,5 @@
...
-AR60-01,,,,GI05MOAS-GL537-01-FLORDM000,3,1,2021-08-15T00:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00537,CGINS-FLORDM-03582,59.925833,-39.299167,,1000,1000,,,
-AR60-01,,,,GI05MOAS-GL537-02-DOSTAM000,3,1,2021-08-15T00:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00537,CGINS-DOSTAM-00401,59.925833,-39.299167,,1000,1000,,,
-AR60-01,,,,GI05MOAS-GL537-04-CTDGVM000,3,1,2021-08-15T00:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00537,CGINS-CTDGVM-09237,59.925833,-39.299167,,1000,1000,,,
\ No newline at end of file
+AR60-01,,,,GI05MOAS-GL537-00-ENG000000,3,1,2021-08-11T04:10:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00537,CGCON-GLDROO-00537,59.85908,-39.12473,,1000,1000,,,
+AR60-01,,,,GI05MOAS-GL537-01-FLORDM000,3,1,2021-08-11T04:10:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00537,CGINS-FLORDM-03582,59.85908,-39.12473,,1000,1000,,,
+AR60-01,,,,GI05MOAS-GL537-02-DOSTAM000,3,1,2021-08-11T04:10:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00537,CGINS-DOSTAM-00401,59.85908,-39.12473,,1000,1000,,,
+AR60-01,,,,GI05MOAS-GL537-04-CTDGVM000,3,1,2021-08-11T04:10:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00537,CGINS-CTDGVM-09237,59.85908,-39.12473,,1000,1000,,,
```
Comment (rtravis-whoi) [inline deployment/GI05MOAS-GL537_Deploy.csv:5]: updated lat looks good

**Review by rtravis-whoi — APPROVED** [review]

**Review by snwhite — APPROVED** [review]

**Pull request merged**